proposals/Level-1.md

```diff
@@ -51,13 +51,12 @@ exceptions.  For embedders that have garbage collection (such as JavaScript),
 this is not a problem.
 
 However, not all embedders may have a garbage collector. For this reason,
-WebAssembly exceptions are designed to allow the use of reference counters (to
-exceptions) to perform the the garbage collection in the embedder.
+WebAssembly exceptions are designed to allow the use of reference counters to
+perform the the garbage collection in the embedder.
```

Comment:
nit: 'the the' -> 'the'

I might rephrase this as "WebAssembly exceptions are designed to allow other storage management methods, such as reference counting, to perform the garbage collection in the embedder." I don't know what other methods their might be, but the idea is tracing GC is not required, using reference counting is an existence proof of this fact, but other methods are allowed too. We don't make any requirements on how the embedder manages storage, we simply say that it's possible to be done.